cmd/tdexd/main.go

```diff
@@ -103,11 +102,20 @@ func main() {
 		log.Errorf("error while setting up webhook pubsub service: %s", err)
 		return
 	}
+
+	network, err := config.GetNetwork()
+	if err != nil {
+		crawlerSvc.Stop()
+		repoManager.Close()
+
+		log.Errorf("error while setting up network: %s", err)
+		return
+	}
+
```

Comment:
instead of doing this here, we should check for the error in [validate()](https://github.com/tdex-network/tdex-daemon/blob/master/config/config.go#L202) func (right after the datadir check) which is called by init(). This way, here we could just skip checking for error.

Reply:
this is cosmetic stuff and IMHO it is more readable if err is handled in this location and not skipped.